src/styles/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border-radius: 5px;
+  font-size: 11px;
+  &__img {
+    width: 160px;
+    box-sizing: content-box;
+    height: 134px;
+    margin: 0 5px 42px;
+    padding-top: 28.5px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    margin-bottom: 5px;
+    margin-block-start: 0;
```

не уверен что правильно использовать margin-block-start: 0 для сброса марджина. Можно было написать margin в одну строку. Или сбросить для всех елементов h2 в reset.css
```suggestion
    margin: 0 0 5px;
```

---

Yes. Very good idea